Doc/library/time.rst

```diff
@@ -19,9 +19,16 @@ An explanation of some terminology and conventions is in order.
 
 .. index:: single: epoch
 
-* The :dfn:`epoch` is the point where the time starts.  On January 1st of that
-  year, at 0 hours, the "time since the epoch" is zero.  For Unix, the epoch is
-  1970.  To find out what the epoch is, look at ``gmtime(0)``.
+* The :dfn:`epoch` is the point where the time starts, and is platform
+  dependent.  For Unix, the epoch is January 1, 1970, 00:00:00 (UTC).
+  To find out what the epoch is on a given platform, look at ``gmtime(0)``.
```

Comment:
I changed it to `time.gmtime(0)` where it first appears in d387bf7. However, when the statement is repeated in the documentation of `time.time`, I retain `gmtime(0)` as this appears to be the convention used in all the function docs for the module.